qa/L0_model_config/autofill_noplatform_success/tensorrt/empty_config/expected

```diff
@@ -41,3 +41,4 @@ optimization {
     enable: true
   }
 }
+backend: "tensorrt"
```

Comment:
If server core builds with `TRITON_ENABLE_TENSORRT`, server-side autofill should be able to set `platform` / `backend` for it as minimum requirement (avoid breaking autofill on old style model config), then the rest of autofill is delegated to TRT backend.

Reply:
I don't understand this clearly. The server-side autofill indeed set the backend field with value "tensorrt".
These are the expected config files that would be compared with the model config returned by the server.

Reply:
Ah, I misread it..